text/setup.py

```diff
@@ -22,9 +22,24 @@
     'scipy',
     'pandas',
     'scikit-learn',
-
+    'fairscale',
+    'scikit-image',
+    'boto3',
+    'pytest',
+    'smart_open',
+    'pytorch_lightning>=1.5.9',
+    'torchmetrics',
+    'transformers>=4.16.2',
+    'Pillow',
+    'tqdm',
+    'pandas',
+    'nptyping',
+    'omegaconf',
+    'timm',
+    'pytest',
+    'sentencepiece',
     f'autogluon.core=={version}',
-    'autogluon-contrib-nlp==0.0.1b20210201',
+    # 'autogluon-contrib-nlp==0.0.1b20210201',
```

Comment:
Will try to work to upgrade `autogluon-contrib-nlp`.

Reply:
Won't this break MXNet version of code if it is commented out?

Reply:
I've updated `autogluon-contrib-nlp`. It can be resolved via

```python
' autogluon-contrib-nlp==0.0.1b20220208',
```